<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmWaveBar</class>
 <widget class="QWidget" name="frmWaveBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="WaveBar" name="widget1"/>
   </item>
   <item>
    <widget class="WaveBar" name="widget2">
     <property name="barColor">
      <color>
       <red>255</red>
       <green>107</green>
       <blue>107</blue>
      </color>
     </property>
    </widget>
   </item>
   <item>
    <widget class="WaveBar" name="widget3">
     <property name="barColor">
      <color>
       <red>24</red>
       <green>189</green>
       <blue>155</blue>
      </color>
     </property>
    </widget>
   </item>
   <item>
    <widget class="WaveBar" name="widget4">
     <property name="barColor">
      <color>
       <red>255</red>
       <green>85</green>
       <blue>255</blue>
      </color>
     </property>
    </widget>
   </item>
   <item>
    <widget class="WaveBar" name="widget5">
     <property name="barColor">
      <color>
       <red>255</red>
       <green>0</green>
       <blue>127</blue>
      </color>
     </property>
    </widget>
   </item>
   <item>
    <widget class="WaveBar" name="widget6">
     <property name="barColor">
      <color>
       <red>0</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </property>
    </widget>
   </item>
   <item>
    <widget class="WaveBar" name="widget7">
     <property name="barColor">
      <color>
       <red>255</red>
       <green>255</green>
       <blue>0</blue>
      </color>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WaveBar</class>
   <extends>QWidget</extends>
   <header>wavebar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
